UniversalAlgebra/Varieties/Basic.lagda

```diff
@@ -167,37 +168,37 @@ Identities modeled by an algebra `𝑨` are also modeled by every subalgebra of
 
 \begin{code}
 
-module _ {α β : Level} {X : Type 𝓧} where
- open ≡-Reasoning
+module _ (wd : SwellDef){𝓤 𝓦 : Level}{X : Type 𝓧} where
```

Comment:
Oh, that's very shifty, taking local `funext` assumptions away to replace them by a module-global one instead!  It does clear the code up nicely, I must admit.

Reply:
Actually, it's maybe not quite as shifty as it looks.  First of all SwellDef is weaker (I believe) than funext.  Secondly, wd must be applied locally, where needed, and must be mentioned explicitly when used.

Reply:
I still don't have a proof that swelldef implies funext.  If you can prove that, please let me know!

Reply:
I'll keep it as a puzzle.